accelerator/config.c

```diff
@@ -46,11 +156,32 @@ status_t ta_config_init(ta_config_t* const info, iota_config_t* const tangle,
   pow_init();
 
   log_info(logger_id, "Initializing cache connection\n");
-  cache_init(REDIS_HOST, REDIS_PORT);
-
+  cache_init(cache->host, cache->port);
   return ret;
 }
 
+void ta_config_print(ta_config_t* const info, iota_config_t* const tangle,
+                     ta_cache_t* const cache,
+                     iota_client_service_t* const service) {
+  printf("TA settings:\n");
+  printf("\t* TA host: %s\n", info->host);
+  printf("\t* TA port: %s\n", info->port);
+  printf("\t* TA thread: %d\n", info->thread_count);
+
+  printf("Redis settings:\n");
+  printf("\t* Redis host: %s\n", cache->host);
+  printf("\t* Redis port: %d\n", cache->port);
+
+  printf("IRI settings:\n");
+  printf("\t* milestone depth: %u\n", tangle->milestone_depth);
+  printf("\t* mwm: %u\n", tangle->mwm);
+  printf("\t* seed: %s\n", tangle->seed);
```

Comment:
And it's dangerous showing sensitive info like seed.

Reply:
I'm not sure it's suitable or not to print out all the config values after setting them :thinking: 
This function can be removed if you think this is not necessary.

Reply:
Let's remove it.